### PR TITLE
Added download links to the workspace invite message

### PR DESCRIPTION
**Pull request by akshayasalvi**
### Details

- Added app download page link to the Workspace invite page

### Fixed Issues

$ https://github.com/Expensify/App/issues/5774

### Tests
1. Tested the email invite from all platforms
2. Tested email link redirection
3. Edited message content removing link

### QA Steps
1. Go to any workspace using sidebar
2. Click on Manage Members
3. Click on Invite button at the top
4. The default welcome note should have `use.expensify.com/download` link in the message 
5. Enter any email address and click on Invite
6. You should also receive the email with the `use.expensify.com/download` as a clickable link

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

email-screenshot

web-app-links

**v2**
image

#### Mobile Web

mweb-app-links

#### Desktop

desktop-app-links

#### iOS

ios-app-links

#### Android

android-app-links

### CODE REVIEW

File: src/languages/en.js [hunk 1]
```diff
@@ -727,7 +727,7 @@ export default {
             personalMessagePrompt: 'Add a personal message (optional)',
             pleaseSelectUser: 'Please select a user from contacts.',
             genericFailureMessage: 'An error occurred inviting the user to the workspace, please try again.',
-            welcomeNote: ({workspaceName}) => `You have been invited to ${workspaceName || 'a workspace'}! Download the Expensify mobile app to start tracking your expenses.`,
+            welcomeNote: ({workspaceName}) => `You have been invited to ${workspaceName || 'a workspace'}! Download the Expensify mobile app using use.expensify.com/download to start tracking your expenses.`,
```
Comment (Beamanator) [inline src/languages/en.js:730]:
```suggestion
            welcomeNote: ({workspaceName}) => `You have been invited to ${workspaceName || 'a workspace'}! Download the Expensify mobile app at use.expensify.com/download to start tracking your expenses.`,
```

**Review by Beamanator — CHANGES_REQUESTED** [review]
Can you actually make this tiny text change? (also in the spanish translation) I think it's a bit clearer to users

**Comment by akshayasalvi** [comment]
PR updated

**Review by Beamanator — APPROVED** [review]
Looks great, nice work - thanks for sticking with this 👍

**Pull request merged**